plugins/kjob.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: krew.googlecontainertools.github.com/v1alpha2
+kind: Plugin
+metadata:
+  name: kjob
+spec:
+  version: v0.1.0
+  shortDescription: Tool for CLI-loving ML researchers.
+  homepage: https://github.com/kubernetes-sigs/kjob/
+  description: |
+    The kubectl-kjob plugin, kjobctl, allows you to list, describe and create jobs.
```

Comment:
Please tell users something more useful. I wouldn't install this since "kubectl create job" command already exists.

Reply:
+1, we should also update the description on the page

Reply:
Maybe something like this:
```
This is a CLI tool implemented as a kubectl plugin. It allows ML researchers to run templated Jobs with different values for the application-specific parameters, without the need to edit and submit entire YAML files. The tool comes with built-in support for running slurm scripts inside a Kubernetes cluster.
```
IIUC, we don't think we need to draw attention to capabilities such as listing or deleting Jobs here, because these are easy with kubectl. The value added is the running of jobs based on the pre-defined templates, allowing the ML researchers to run the same job with different values of the application-specific parameters.

Reply:
Thank you so much! Done.

Reply:
https://github.com/kubernetes-sigs/kjob/pull/41